docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -136,12 +137,26 @@ If you need a feature from a preview release (that isn't yet available in a stab
 
 Yugabyte manages upgrades for you. After you choose a track, database upgrades continue to take releases from the track you chose. For multi-node clusters, Yugabyte performs a rolling upgrade without any downtime. You can manage when Yugabyte performs maintenance and upgrades by configuring the [maintenance window](../../cloud-clusters/cloud-maintenance/) for your cluster.
 
-<!-- #### Staging clusters
+### Staging cluster
 
-Yugabyte tests every version in the stable branch for backwards compatibility. However, it's good practice to first test database updates against your pre-production environment (aka development, testing, staging, or canary environment) to ensure compatibility before upgrading your production clusters.
+Use a staging cluster for the following tasks:
 
-Create a staging cluster (this can be smaller than your production cluster) and configure your pre-production environment to connect to it. When you are notified of an upcoming maintenance event, schedule the [maintenance windows](../../cloud-clusters/cloud-maintenance/) for the staging and production cluster so that you can validate updates against your applications in your pre-production environment before updating your production cluster.
--->
+- Verifying your application is compatible with [database updates](#database-updates).
+- Ensuring your application correctly handles a rolling restart of the database without errors.
+- Testing new features. Use your staging (aka development, testing, pre-production, or canary) environment to try out new database features while your production systems are still running a previous version.
+- Testing scaling operations and disaster recovery. Find out how your environment responds to a scaling operation, outages, or the loss of a node.
+
+Create a staging cluster and configure your staging environment to connect to it. The staging cluster can be smaller than your production cluster, but ensure it has enough resources and capacity to handle a reasonable load. Sandbox clusters are too [resource-limited](../create-clusters/create-clusters-free/#limitations) for staging.
+
+#### Database upgrades
+
+Yugabyte tests every version in the stable branch for backwards compatibility. However, before upgrading your production cluster, it's good practice to first test your pre-production environment against database updates to ensure your applications are compatible. You want to make sure that an update doesn't have any performance impact on your application. For example, new functionality or a change in the optimizer could impact performance for a single query in a complex application.
```

Comment:
Suggestion:
Yugabyte tests every version in the stable branch for backwards compatibility.  >  Every YugabyteDB version in the stable branch is tested for backwards compatibility.